internal/storage/postgreSQL/storage.go

```diff
@@ -0,0 +1,45 @@
+package postgreSQL
```

Comment:
I would name it only `postgres`
```suggestion
package postgres
```